packages/text-bitmap/src/BitmapText.ts

```diff
@@ -1,29 +1,49 @@
-import { Container } from '@pixi/display';
 import { ObservablePoint, Point } from '@pixi/math';
 import { settings } from '@pixi/settings';
-import { Sprite } from '@pixi/sprite';
+import { Mesh, MeshGeometry, MeshMaterial } from '@pixi/mesh';
```

Comment:
Could you make sure you tweak the dependencies in the package.json of text-bitmap to reflect this change?